addon/__test__/options.test.jsx

```diff
@@ -0,0 +1,191 @@
+/* eslint-disable */
```

Comment:
Can we put the tests in either `addon/test` or maybe `test` in the top level-directory? I think I'd prefer that to having the underscores.

Also, I'd like ESLint enabled here if possible - when we've been enabling it in Firefox we found lots of places where tests weren't running correctly or as expected. Turning ESLint on helped to show those up.

Reply:
By default Jest scans for `__test__` directories. I believe this can be changed by adding a jest config file, but it would be non-standard. Would you like me to look into that?